src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -810,7 +829,9 @@ def __init__(self, config: PretrainedConfig):
 
         self.distilbert = DistilBertModel(config)
         self.qa_outputs = nn.Linear(config.dim, config.num_labels)
-        assert config.num_labels == 2
+        if config.num_labels != 2:
+            raise ValueError("config.num_labels should be 2, but it is " + str(config.num_labels))
```

Comment:
```suggestion
            raise ValueError("config.num_labels should be 2, but it is {config.num_labels)}"
```

Reply:
I think you missed the opening parenthesis, but I sincerely appreciate this suggestion!

Reply:
Ah yes you are right I missed the parenthesis here ! :)